crypto/storage/fs.go

```diff
@@ -124,32 +125,26 @@ func (fsc *fileSystemBackend) SavePrivateKey(kid string, key crypto.PrivateKey)
 	}
 
 	_, err = outFile.Write([]byte(pem))
-	if err != nil {
-		return err
-	}
 
-	return fsc.SavePublicKey(kid, key)
+	return err
 }
 
 // Save the private key for the given key to disk. Files are  postfixed with '_private.pem'. Keys are stored in pem format. It also store the public key
```

Comment:
although already present before this PR, godoc is incorrect

Reply:
though I fixed that somewhere....